<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IGBPCA6" failures="3" tests="7" name="Test" time="28.618" errors="0" timestamp="2024-10-29T13:21:26 IST">
  <testcase classname="api.test.AgencyTest" name="CreateAgency" time="17.058">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.AgencyTest.CreateAgency(AgencyTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- CreateAgency -->
  <testcase classname="api.test.AgencyTest" name="GetAgencyById" time="0.801">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at api.test.AgencyTest.GetAgencyById(AgencyTest.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- GetAgencyById -->
  <testcase classname="api.test.LeadTest" name="GetLeadById" time="3.503"/>
  <testcase classname="api.test.AgentTest" name="CreateAgent" time="3.465">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at api.test.AgentTest.CreateAgent(AgentTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- CreateAgent -->
  <testcase classname="api.test.AgentTest" name="GetAgentById" time="0.725"/>
  <testcase classname="api.test.BranchTest" name="CreateBranch" time="1.247"/>
  <testcase classname="api.test.BranchTest" name="GetBranchByID" time="0.909"/>
</testsuite> <!-- Test -->
